coriolis/osmorphing/debian.py

```diff
@@ -2,9 +2,22 @@
 # All Rights Reserved.
 
 import os
+import yaml
 
```

Comment:
NIT: `io` import should be grouped with the other builtins above (i.e. this newline is extraneous)